articles/install-guide/csinstall.md

```diff
@@ -135,7 +136,86 @@ Of course, you can also build and run Q# programs from the command line by simpl
 
         You should see the following output: `Hello quantum world!`
 
+## Q# with other .NET languages <a name="dotnet"></a>
+
+Q# is compiled to C# for the purposes of local simulation. This makes them very easy to combine. However, you can also use other .NET languages like F# or VB\.NET, although the process is less straightforward.
+
+### Develop with Q# + F# creating a Q# library
+
```

Comment:
We probably need to link to https://github.com/microsoft/Quantum/tree/master/samples/interoperability/fsharp/ somewhere in this section? I've found that having an end-to-end example is very helpful when trying to reproduce it.